frontend/src/metabase/containers/ItemPicker/ItemPicker.unit.spec.js

```diff
@@ -95,8 +94,7 @@ async function setup({
   extraCollections = [],
   ...props
 } = {}) {
-  setupCollectionItemsEndpoint({ dashboards: Object.values(DASHBOARD) });
-  setupRootCollectionItemsEndpoint({
+  setupCollectionItemsEndpoint({
     collections: Object.values(COLLECTION).concat(extraCollections),
```

Comment:
`Object.values().concat` is not only old school, but also a more performant version 😸